2021-11-23 13:06:43,624	INFO scripts.py:670 -- Local node IP: 192.33.123.123
Traceback (most recent call last):
  File "/work/xchang/anaconda3/envs/CQR/bin/ray", line 33, in <module>
    sys.exit(load_entry_point('ray==1.4.0', 'console_scripts', 'ray')())
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/ray/scripts/scripts.py", line 1808, in main
    return cli()
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/ray/scripts/scripts.py", line 679, in start
    ray_params, head=False, shutdown_at_exit=block, spawn_reaper=block)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/ray/node.py", line 206, in __init__
    "metrics_export_port", default_port=ray_params.metrics_export_port)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/site-packages/ray/node.py", line 635, in _get_cached_port
    ports_by_node.update(json.load(f))
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/json/__init__.py", line 296, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/work/xchang/anaconda3/envs/CQR/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
